pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -337,6 +337,7 @@ public void close() throws PulsarServerException {
             if (closeFuture != null) {
                 return closeFuture;
             }
+            LOG.info("Closing");
```

Comment:
We could use more explicit indication of what we're closing (and the other logs here). I know the logger name will contain the class name, but just to make it more clear and upfront.

```suggestion
            LOG.info("Closing PulsarService");
```

Reply:
Applied. Thanks!